openvdb_houdini/SOP_NodeVDB.cc

```diff
@@ -686,6 +698,39 @@ OpenVDBOpFactory::OpenVDBOpFactory(
     houdini_utils::OpFactory::OpFlavor flavor):
     houdini_utils::OpFactory(OpenVDBOpPolicy(), english, ctor, parms, table, flavor)
 {
+    mNativeName = OpenVDBOpPolicy().getNativeName(*this);
+}
+
+
+OpenVDBOpFactory::~OpenVDBOpFactory()
+{
+    // hide the native node if marked invisble
+    if (!mNativeName.empty()) {
+        bool invisible = mNativeInvisible;
+
```

Comment:
Maybe collapse this to
```
if (mNativeInvisible && !mNativeName.empty()) {
    this->table().addOpHidden(mNativeName.c_str());
}
```
(Testing the boolean first is faster.)

Reply:
This isn't in the hot code path, so I don't think it's necessary to optimize this. The main reason I did it like this is so that changing the logic to adjust the invisible variable can be done without modifying any existing lines. Adding, reverting and combining changes as we figure out the right logic here should be less likely to go wrong as a result.